locals.tf

```diff
@@ -1,6 +1,6 @@
 locals {
   region          = "us-west-2"
   app-name        = "learning-project"
-  master-password = "1qaz!QAZ"
+  master_password = "1qaz!QAZ"
```

Comment:
be consistent in naming, if you use `_` for `master_password `, then please use `_` for `app_name`